condorde: qsopt compilation issue on some macs

```
$ make -C bin/
make: Entering directory '/Users/karel/git/ism-supplement/bin'
./install_concorde.sh
Downloading QSopt for Darwin x86_64 ...
Downloading Concorde source ...
Extracting Concorde ...
Configuring Concorde ...
creating cache ./config.cache
checking host system type... Invalid configuration `darwin': machine `darwin' not recognized

checking for prespecified compiler options... no
checking for gcc... gcc
checking whether the C compiler (gcc -fPIC -O3 -g -std=gnu89 ) works... yes
checking whether the C compiler (gcc -fPIC -O3 -g -std=gnu89 ) is a cross-compiler... no
checking whether we are using GNU C... yes
checking whether gcc accepts -g... yes
checking whether make sets ${MAKE}... yes
checking for ranlib... ranlib
checking for sh... /bin/sh
checking for file suffixes... o, a, 
checking for sqrt in -lm... yes

...

a - mincut.o
a - cut_st.o
a - gomoryhu.o
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/COMBS'
ranlib heldkarp.a
ranlib util.a
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TINY'
ranlib cut.a
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LP'
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/HELDKARP'
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/CUT'
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
gcc -fPIC -O3 -g -std=gnu89  -o localcut loc_main.o localcut.a ../TSP/tsp.a           ../PQ/pq.a ../EDGEGEN/edgegen.a   ../LINKERN/linkern.a ../VERIFY/verify.a     ../FMATCH/fmatch.a ../KDTREE/kdtree.a     ../COMBS/combs.a ../TINY/tiny.a         ../LP/lp.a ../BIGGUY/bigguy.a     ../UTIL/util.a ../HELDKARP/heldkarp.a ../CUT/cut.a ../TSP/tsp.a           ../CUT/cut.a /var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T//concorde-build.oyT2xF/qsopt/qsopt.a -lm 
ld: warning: ignoring duplicate libraries: '../CUT/cut.a', '../TSP/tsp.a'
ld: unknown file type in '/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T//concorde-build.oyT2xF/qsopt/qsopt.a'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [Makefile:60: localcut] Error 1
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LOCALCUT'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/COMBS'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/COMBS'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/CUT'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o mc_main.o mc_main.c
cd ../UTIL; make util.a
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
gcc -fPIC -O3 -g -std=gnu89  -o mincut mc_main.o cut.a ../UTIL/util.a -lm 
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/CUT'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o eg_main.o eg_main.c
cd ../KDTREE; make kdtree.a
cd ../FMATCH; make fmatch.a
cd ../LINKERN; make linkern.a
cd ../UTIL; make util.a
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: 'kdtree.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: 'fmatch.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: 'linkern.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
gcc -fPIC -O3 -g -std=gnu89  -o edgegen eg_main.o edgegen.a ../KDTREE/kdtree.a   ../FMATCH/fmatch.a ../LINKERN/linkern.a ../UTIL/util.a -lm 
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o fm_main.o fm_main.c
cd ../EDGEGEN; make edgegen.a
cd ../KDTREE; make kdtree.a
cd ../LINKERN; make linkern.a
cd ../UTIL; make util.a
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: 'edgegen.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: 'kdtree.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: 'linkern.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
gcc -fPIC -O3 -g -std=gnu89  -o fmatch fm_main.o fmatch.a ../EDGEGEN/edgegen.a ../KDTREE/kdtree.a ../LINKERN/linkern.a ../UTIL/util.a -lm 
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/HELDKARP'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/HELDKARP'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o lk_main.o lk_main.c
cd ../EDGEGEN; make edgegen.a
cd ../KDTREE; make kdtree.a
cd ../FMATCH; make fmatch.a
cd ../UTIL; make util.a
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: 'kdtree.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: 'edgegen.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: 'fmatch.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
gcc -fPIC -O3 -g -std=gnu89  -o linkern lk_main.o linkern.a ../EDGEGEN/edgegen.a ../KDTREE/kdtree.a ../FMATCH/fmatch.a   ../UTIL/util.a -lm 
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LP'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LP'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/PQ'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/PQ'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TINY'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TINY'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TOOLS'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o showres.o showres.c
cd ../KDTREE; make kdtree.a
cd ../UTIL; make util.a
cd ../EDGEGEN; make edgegen.a
cd ../LINKERN; make linkern.a
cd ../FMATCH; make fmatch.a
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o tourchk.o tourchk.c
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o tourlen.o tourlen.c
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o prob2tsp.o prob2tsp.c
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o edg2len.o edg2len.c
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o edgunion.o edgunion.c
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o fconvert.o fconvert.c
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: 'kdtree.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: 'linkern.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: 'fmatch.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: 'edgegen.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
gcc -fPIC -O3 -g -std=gnu89  -o edgunion edgunion.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
gcc -fPIC -O3 -g -std=gnu89  -o tourchk tourchk.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
gcc -fPIC -O3 -g -std=gnu89  -o prob2tsp prob2tsp.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
gcc -fPIC -O3 -g -std=gnu89  -o tourlen tourlen.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
gcc -fPIC -O3 -g -std=gnu89  -o edg2len edg2len.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
gcc -fPIC -O3 -g -std=gnu89  -o fconvert fconvert.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
gcc -fPIC -O3 -g -std=gnu89  -o showres showres.o ../KDTREE/kdtree.a ../UTIL/util.a ../EDGEGEN/edgegen.a ../LINKERN/linkern.a ../FMATCH/fmatch.a ../KDTREE/kdtree.a ../UTIL/util.a -lm 
ld: warning: ignoring duplicate libraries: '../KDTREE/kdtree.a', '../UTIL/util.a'
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TOOLS'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TSP'
gcc -fPIC -O3 -g -std=gnu89  -I../INCLUDE -I../INCLUDE   -c -o concorde.o concorde.c
cd ../PQ; make pq.a
cd ../VERIFY; make verify.a
cd ../LOCALCUT; make localcut.a
cd ../TINY; make tiny.a
cd ../LINKERN; make linkern.a
cd ../EDGEGEN; make edgegen.a
cd ../KDTREE; make kdtree.a
cd ../FMATCH; make fmatch.a
cd ../UTIL; make util.a
cd ../BIGGUY; make bigguy.a
cd ../CUT; make cut.a
cd ../HELDKARP; make heldkarp.a
cd ../LP; make lp.a
cd ../COMBS; make combs.a
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/PQ'
make[3]: 'pq.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/PQ'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TINY'
make[3]: 'tiny.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TINY'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/VERIFY'
make[3]: 'verify.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/VERIFY'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LOCALCUT'
make[3]: 'localcut.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LOCALCUT'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: 'linkern.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LINKERN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: 'edgegen.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/EDGEGEN'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: 'kdtree.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/KDTREE'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: 'util.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: 'fmatch.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/FMATCH'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/BIGGUY'
make[3]: 'bigguy.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/BIGGUY'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/CUT'
make[3]: 'cut.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/CUT'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/HELDKARP'
make[3]: 'heldkarp.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/HELDKARP'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LP'
make[3]: 'lp.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/LP'
make[3]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/COMBS'
make[3]: 'combs.a' is up to date.
make[3]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/COMBS'
gcc -fPIC -O3 -g -std=gnu89  -o concorde concorde.o tsp.a ../PQ/pq.a             ../VERIFY/verify.a ../LOCALCUT/localcut.a   ../TINY/tiny.a ../LINKERN/linkern.a   ../EDGEGEN/edgegen.a ../KDTREE/kdtree.a     ../FMATCH/fmatch.a ../UTIL/util.a ../BIGGUY/bigguy.a     ../CUT/cut.a ../HELDKARP/heldkarp.a ../LP/lp.a ../COMBS/combs.a /var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T//concorde-build.oyT2xF/qsopt/qsopt.a -lm 
ld: unknown file type in '/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T//concorde-build.oyT2xF/qsopt/qsopt.a'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [Makefile:58: concorde] Error 1
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/TSP'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/UTIL'
make[2]: Entering directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/VERIFY'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde/VERIFY'
make[1]: Leaving directory '/private/var/folders/3d/rt3z63015n56plmzcckx34f00000gn/T/concorde-build.oyT2xF/src/concorde'
Build finished, but TSP/concorde was not produced.
make: *** [Makefile:16: concorde] Error 1
make: Leaving directory '/Users/karel/git/ism-supplement/bin'
```

--- a/bin/install_concorde.sh
+++ b/bin/install_concorde.sh
@@ -64,7 +64,7 @@ case "${OS}:${ARCH}" in
     CONFIGURE_EXTRA=(--host=darwin)
     ;;
   Darwin:x86_64)
-    QSOPT_A_URL="https://www.math.uwaterloo.ca/~bico/qsopt/downloads/codes/mac64/qsopt.a"
+    QSOPT_A_URL="https://www.math.uwaterloo.ca/~bico/qsopt/downloads/codes/mac64/qsopt.a.gz"
     QSOPT_H_URL="https://www.math.uwaterloo.ca/~bico/qsopt/downloads/codes/mac64/qsopt.h"
     CONFIGURE_EXTRA=(--host=darwin)
     ;;
@@ -100,7 +100,15 @@ SRC_DIR="$WORK_ROOT/src"
 mkdir -p "$QSOPT_DIR" "$SRC_DIR"
 
 echo "Downloading QSopt for ${OS} ${ARCH} ..."
-fetch "$QSOPT_A_URL" "$QSOPT_DIR/qsopt.a"
+if [[ "$QSOPT_A_URL" == *.gz ]]; then
+ fetch "$QSOPT_A_URL" "$QSOPT_DIR/qsopt.a.gz"
+ gunzip -c "$QSOPT_DIR/qsopt.a.gz" > "$QSOPT_DIR/qsopt.a"
+else
+ fetch "$QSOPT_A_URL" "$QSOPT_DIR/qsopt.a"
+fi
+
+file "$QSOPT_DIR/qsopt.a"
+ar -t "$QSOPT_DIR/qsopt.a" >/dev/null
 fetch "$QSOPT_H_URL" "$QSOPT_DIR/qsopt.h"
 
 echo "Downloading Concorde source ..."

